WordPress/Classes/ViewRelated/Notifications/Comment Moderation/CommentModerationViewModel.swift

```diff
@@ -63,11 +82,176 @@ final class CommentModerationViewModel: ObservableObject {
     }
 
     func didTapLike() {
-        switch state {
-        case .approved(let liked):
-            state = .approved(liked: !liked)
-        default:
-            break
+        let initialIsLiked = comment.isLiked
+
+        guard let siteID = siteID else {
+            self.state = .approved(liked: initialIsLiked)
+            return
+        }
+
+        if initialIsLiked {
+            track(withEvent: .notificationsCommentLiked) { comment in
+                CommentAnalytics.trackCommentUnLiked(comment: comment)
+            }
```

Comment:
[Suggestion] The event should be `notificationsCommentUnliked`

Reply:
[85386dd](https://github.com/wordpress-mobile/WordPress-iOS/pull/23225/commits/85386dd8a130fe5f81aa99d2ca283e3b9d71c938)